catalog/opensearch/catalog-opensearch-endpoint/src/test/java/org/codice/ddf/opensearch/query/OsNamespaceContext.java

```diff
@@ -54,7 +52,7 @@ public String getPrefix(String namespaceURI) {
 
   @Override
   public Iterator<String> getPrefixes(String namespaceURI) {
-    ArrayList<String> prefixList = new ArrayList<String>(1);
+    ArrayList<String> prefixList = new ArrayList<>(1);
```

Comment:
Having the size at `1` is going to cause the add to resize the list every single time.
How about just `return Collections.singletonList(getPrefix(namespaceURI)).iterator()`?